syft/federated/plan.py

```diff
@@ -90,9 +99,9 @@ def __init__(
 
         # Info about the plan stored
         self.plan = list()
-        self.readable_plan = list()
-        self.arg_ids = list()
-        self.result_ids = list()
+        self.readable_plan = readable_plan if readable_plan else []
```

Comment:
Could you add an is None here. Technically everything with a false truth value would be converted to [] here